build-manual.py

```diff
@@ -73,6 +73,78 @@ def __init__(self, number: str, title: str, filename: str):
 		self.filename = filename
 		self.items = []
 
+def make_one_page(dest_directory):
+	"""
+	Generate one-page php file of the manual.
+	"""
+	# Get all php files in destination directory.
+	php_files = list(filter(lambda x: x.endswith("php"), os.listdir(dest_directory)))
+	php_re = regex.compile(r"<\?.+?\?>", regex.S)
+	index = php_files.pop(php_files.index("index.php"))
+
+	with open(dest_directory + "/" + index, "r", encoding="utf-8") as f:
```

Comment:
Please use `pathlib` to create paths instead of string concatenation: https://docs.python.org/3/library/pathlib.html